src/resolvers/group.js

```diff
@@ -79,6 +80,23 @@ export default {
         context,
         requestInfo
       )
+    },
+
+    acls: async (source, args, context, info) => {
+      const { groupId } = source
+
+      const { dataSources } = context
+
+      const a = await dataSources.aclSourceFetch(
```

Comment:
Here I think we can just return the dataSource, don't think a `const` is needed. 
 ```
return dataSources.aclSourceFetch(
   handlePagingParams({
     ...args,
     permittedGroup: groupId
   }),
   context,
   parseResolveInfo(info)
 ) 
```

Reply:
Oh man, good catch, I was testing something.